dali/pipeline/operator/arg_helper.h

```diff
@@ -59,6 +59,18 @@ struct ArgShapeFromSize<1> {
  * @}
  */
 
+
+/**
+ * @brief ArgValue flags for acquire
+ *
+ */
+enum ArgValueFlags : unsigned {
+  ArgValue_EnforceUniform = 0b0001,
+  ArgValue_AllowEmpty = 0b0010,
+  ArgValue_Default = 0
+};
```

Comment:
To consider: put `ArgValue` and drop `ArgValue_` prefix?

Reply:
Also:
```C++
constexpr Flags operator |(Flags a, Flags b) {
    return Flags(int(a)|int(b)); // NOLINT
}
```
This way you can pass `Flags` instead of `unsigned int` to `Acquire`.

Reply:
I prefer not to put it inside the scope of ArgValue because that's a template.
I'll take the `operator|` suggestion